features/items.md

```diff
@@ -167,18 +184,17 @@ The output will be
     Window is 1
     Window is opened
 
-Transform file use ISO-8859-1 encoding. If you want your accent and special character to show in the right way, you need to precise this encoding in your editor. If you must use UTF-8, you will need to escape the character. See [UTF-8 encoding table](http://www.utf8-chartable.de/)
-
-Example: é will be written ```\u00E9```
+Transform files use UTF-8 encoding, so unicode symbols will also work.
 
-If the key contains spaces, you have to escape them with a \
-
-    VALUE\ WITH\ SPACES=value with spaces
+    ARIES=♈ Aries
+    TAURUS=♉ Taurus
+    WAXING_CRESCENT=🌑→🌓 Waxing Crescent
+    FIRST_QUARTER=🌓 First Quarter
 
 ## Icons
-OpenHAB provides you a set of basic icons by default. However if you wish to use custom icons you need to create the folder ```${openhab_home}/conf/icons/classic/``` and place all your icons inside. These icons will be used in all of the openHAB frontends. The images must be in .png or .svg format, and have a name with only small letters and a hyphen or underscore (if required). The PaperUI interface (or via the classicui.cfg or basicui.cfg files) allows you to define whether you use Vector (.svg) or Bitmap (.png) icon files.
+OpenHAB provides you a set of [basic icons](http://docs.openhab.org/addons/iconsets/classic/readme.html) by default. However if you wish to use custom icons you need to create the folder `conf/icons/classic/` and place all your icons inside. These icons will be used in all of the openHAB frontends. The images must be in .png or .svg format, and have a name with only small letters and a hyphen or underscore (if required). The PaperUI interface (or via the classicui.cfg or basicui.cfg files) allows you to define whether you use Vector (.svg) or Bitmap (.png) icon files.
```

Comment:
The folder will be created for you / will already exist. I provided a PR a few weeks ago exactly with that purpose ;)